bin/scripts/lib/fonts.json

```diff
@@ -1,5 +1,16 @@
 {
   "fonts": [
+    {
+      "unpatchedName": "0xProto",
+      "RFN": false,
+      "patchedName": "0xProto",
+      "folderName": "0xProto",
+      "imagePreviewFont": "0xProto",
+      "linkPreviewFont": "0xProto",
+      "caskName": "0xProto",
+      "repoRelease": true,
```

Comment:
For all new fonts this should be `false`

Reply:
Changed 👍